app/controllers/application_controller.rb

```diff
@@ -1,5 +1,6 @@
 class ApplicationController < ActionController::Base
   include Pundit
+  include BrandedExperience
```

Comment:
i hear you, though i'm partial to the upside you outlined -- going to leave as is for now